index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Survey Form microverse</title>
+</head>
+
+<body>
+    <div id="parent title">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </div>
```

Comment:
- [ ] Always make sure to not give orders, but rather give polite suggestions. It's better to use `Please make sure to use HTML5 semantic tags..` rather than give straight orders, remember, we're all humans! 👍